### DISCUSSION TITLE
[MacOS](Gothic I) Old Mine graphic bug

### DISCUSSION THREAD

**Discussion by ghost** [Try/OpenGothic#506] [General]
Hello,
I still started playing the first Gothic!
But there was a problem associated with the Old Mine.
When I enter it, it is all white (I will attach screenshots below).
But this is half the trouble, you can even put up with it. But here's what you can't do, so it's with the complete loss of the picture. The picture disappears when I go down the Old Mine.

I don't know if you are doing the first Gothic, but I found such a graphical bug.

Screenshots uploaded to Google Drive - https://drive.google.com/drive/folders/14R0w2fH_dhms9pt5pftBt-gQv_5kd1Y1?usp=sharing

**Comment by Try** [comment]
Hello, @xxZen !

I've tested old mine on my PC - no noticeable issues:
![изображение](https://github.com/Try/OpenGothic/assets/122894/6ced1e31-a54c-4f11-ad77-86c7dbc8d509)

Can you provide application logs (`log.txt`)? Thanks!

**Reply by damiel** [reply]
I encountered the same issue today when I tried OpenGothic for the first time(very impressive work btw, thanks a lot for doing this, I had a blast playing these games again :) ), will try and see if I can provide the log instead.

**Reply by damiel** [reply]
Here is my log.txt:

`OpenGothic v
GPU = Apple M1 Pro
Depth format = Depth32F Shadow format = Depth16
[phoenix] World: Parsing object [MeshAndBsp % 0 0]
[phoenix] BspTree: Parsing chunk c000
[phoenix] BspTree: Parsing chunk c010
[phoenix] BspTree: Parsing chunk c040
[phoenix] BspTree: Parsing chunk c045
[phoenix] BspTree: Parsing chunk c050
[phoenix] BspTree: Parsing chunk c0ff
[phoenix] World: Parsing object [VobTree % 0 0]
[phoenix] World: Parsing object [WayNet % 0 0]
[phoenix] World: Parsing object [EndMarker % 0 0]
unable to load sound fx: MINEAMBIENCE1
unable to load sound fx: MINEAMBIENCE2
[phoenix] ModelScript: Syntax error (line 3186, column 2): comments must start with two slashes
comb not found: t_FIREPLACE_Stand_2_S0 -> c_FP_STAND_2_S0_(c_FP_STAND_2_S0_2)
comb not found: t_FIREPLACE_S0_2_Stand -> c_FP_STAND_2_S0_(c_FP_STAND_2_S0_2)
alias not found: t_Walk_2_WalkBL -> t_walkL_2_Walk
[phoenix] ModelScript: Syntax error (line 138, column 55): String not terminated
comb not found: t_FIREPLACE_Stand_2_S0 -> c_FP_STAND_2_S0_(c_FP_STAND_2_S0_2)
comb not found: t_FIREPLACE_S0_2_Stand -> c_FP_STAND_2_S0_(c_FP_STAND_2_S0_2)
[phoenix] World: Parsing object [VobTree % 0 0]
[phoenix] World: Parsing object [EndMarker % 0 0]
not implemented call [NPC_GETNEXTWP]`

For context I am playing:
* the Gothic I Classic version from Steam 
* on a 14-inch, 2021 MacBook Pro with a Apple M1 Pro
* running Ventura 13.6.5
* using following CI build: https://ci.appveyor.com/project/Try/opengothic/builds/49508516/job/heo5bja565lkr43g

Made a safe game close to the spot where it triggered the blackscreen, loaded the safe and quit the game to produce the log above.

If you need more details or if there is anything else I can provide to debug this lemme know please @Try :)
